speed-dreams/source-2.2.3/src/simulatedDrivingAssistance/backend/ConfigEnums.h

```diff
@@ -29,13 +29,16 @@ typedef unsigned int InterventionType;
 /// @brief The different actions that can be performed
 typedef unsigned int InterventionAction;
 
-#define INTERVENTION_ACTION_NONE       0
+#define INTERVENTION_ACTION_STEER_NONE 0
 #define INTERVENTION_ACTION_TURN_LEFT  1
 #define INTERVENTION_ACTION_TURN_RIGHT 2
 #define INTERVENTION_ACTION_BRAKE      3
 #define INTERVENTION_ACTION_ACCELERATE 4
+#define INTERVENTION_ACTION_BRAKE_NONE 5
```

Comment:
You have a STEER_NONE and BRAKE_NONE, why not an ACCELERATE_NONE ?

Reply:
Omdat accelerate en brake allebei onder speed control vallen, dus het is niet logisch om die apart te hebben.